src/components/UploadForm.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+import ProgressBar from './ProgressBar';
+import "../styles/upload-form.css"
+
+const UploadForm = () => {
+  
+
+    const [ file, setFile ] = useState(null);
+    const [ error, setError ] = useState(null);
+    
+    const types = ['image/png', 'image/jpeg']
```

Comment:
I like the use of .includes() instead of a dicey if statement! Very good start.

If you want to make it even cleaner, you can use use the accept prop on the input, and give it a string, like `accept="image/png, image/jpeg"`, and then the HTML does all the validation for you. 

If you want additional validation (which you probably don't at this point), like size, dimension, etc, then that's something I would do programatically.